src/applications/personalization/profile360/containers/PaymentInformation.jsx

```diff
@@ -102,6 +120,56 @@ class PaymentInformation extends React.Component {
     }
   }
 
+  handleDirectDepositUpdateSubmit(data) {
+    this.props.savePaymentInformation(data);
+    recordEvent({
+      event: 'profile-transaction',
+      'profile-section': 'direct-depost-information',
+    });
+  }
+
+  handleEditClick(e) {
+    const gaProfileAction = 'edit-link';
+
+    // Open edit modal.
+    this.props.editModalToggled();
+
+    // Push Google Analytics event
+    switch (e.currentTarget.dataset.editTarget) {
```

Comment:
Cool! But, you should be able to avoid the `switch` statement and pass the value for `profile-section` directly via the `data-edit-target`, which would also remove the need to define these `EDIT_TARGET` constants. But I'd actually change `data-edit-target` to `data-profile-section` on each `<ProfileFieldHeading>`.

Reply:
Switch removed.